pkg/providers/instancetype/zz_generated.bandwidth.go

```diff
@@ -42,6 +42,7 @@ var (
 		"t4g.medium":        256,
 		"m7a.medium":        390,
 		"m7i-flex.large":    390,
+		"r7a.medium":        390,
```

Comment:
I think we can drop updates to the `codegen` that are unrelated to the PR itself.